website/cue/reference/components.cue

```diff
@@ -35,7 +37,9 @@ components: {
 			service_providers: [string, ...string] | *[]
 		}
 
-		stateful: bool
+		if Args.kind != "enrichment_table" {
+			stateful: bool
+		}
```

Comment:
I presume stateful is not a valid concept in enrichment tables. However, not adding the field here causes errors on line 1202 of this file where it attempts to check the value of this field. I'm not sure how to fix those errors.